src/educational-technology.rst

```diff
@@ -1,18 +1,227 @@
+Another is related to being as concrete as possible versus abstract, as people need to understand concrete concepts before they
+can begin to apply abstractions. Additionally, educational programming environments often are designed to give immediate feedback, 
+with one way of accomplishing this being using visual notation that helps the user confirm or refine their beliefs about how the
+program being written behaves.
+
+.. container:: bib-item
+
+  .. bibliography:: educational-technology.bib
+    :filter: key == 'guzdial2004programming'
+
+  This paper gives an overview of a variety of different programming environments for novices. The educational environments are explored
+  under the lense of how they try to answer the question of "What makes programming hard?"
+
+.. container:: bib-item
+
+  .. bibliography:: educational-technology.bib
+    :filter: key == 'kelleher2005lowering'
+
+  This paper also gives an overview of various programming languages and environments that were designed for supporting novice programmers. 
+  These languages and environments are explored under the lense of supporting the different answers to "What is a person's motivation for trying
+  to learn programming?"
+
+Games
+=====
+
+Programming to acheive an objective in a game is a common technique for introducing individuals to programming skills such
+as problem solving. This is especially true when the audience is children, who may need more motivation to engage in these forms of learning activities than adults actively seeking
+out education in programming.
+
+.. container:: bib-item
+
+  .. bibliography:: educational-technology.bib
+    :filter: key == 'fessakis2013problem'
+
+  This paper goes over the effectiveness of integrating a programming activity into a regular kindergarten class using puzzles
+  that give movement directions to a lady bug to reach some designated destination. The students in the class overall had fun
+  solving the puzzles. The children solved the problems in a step-wise fashion, typically only giving the lady bug 1-3 move 
+  commands before running the actions and checking to where the lady bug moved.
+
+.. todo:: 
+    Add other game based programming environments
 
 Block-Based Environments
 ========================
 
+Block-based environments are often used for teaching young students (pre-university) and students who have had little or no previous exposure to programming.
+These environments have the advantage that there is a reduced coginitive load on learners when it comes to syntax. Block-based environments can limit or eliminate syntax errors
+which saves novices time and confusion resolving such errors and allows them to focus on the higher level concepts. Additionally, the use of blocks
+minimizes the demand for typing and shows program construction in a graphical way, which is especially helpful for children.
```

Comment:
you might also want to mention that blocks are familiar to children from other forms of play, e.g. Legos